app/controllers/additionalReference/index/RemoveAdditionalReferenceYesNoController.scala

```diff
@@ -49,7 +48,7 @@ class RemoveAdditionalReferenceYesNoController @Inject() (
 
   private def addAnother(arrivalId: ArrivalId, additionalReferenceIndex: Index, mode: Mode): Call =
     Call("GET", "#")
-  //TODO: replace with AddAnotherAdditionalReferenceController
+  //TODO: replace with
```

Comment:
Can you AddAnotherAdditionalReferenceController route be added here?

Reply:
Done...